### PR TITLE
Add staging project for etcdadm

### PR DESCRIPTION
**Pull request by justinsb**
Per https://github.com/kubernetes/k8s.io/tree/master/k8s.gcr.io#creating-staging-repos

fellow owners: @neolit123 @dlipovetsky

### CODE REVIEW

**Comment by dlipovetsky** [comment]
Thank you @justinsb! This brings etcdadm one step closer to binary releases (https://github.com/kubernetes-sigs/etcdadm/issues/100)

**Comment by stp-ip** [comment]
/hold

This feels like user facing and binary artifacts could have a need to be stored longer or?
Thanks for giving some context on the usage and therefore needed retention.

Wanted to mention that GCS buckets created within the staging repos do not have extended retention. It comes with similar retention promises as staging GCR images aka months not years.

There is a plan for binary artifact promotion, but that's not being worked on/available yet.
For longer retention a production GCS bucket could potentially "manually"  be created similar to https://github.com/kubernetes/k8s.io/issues/654.

**Comment by stp-ip** [comment]
We briefly discussed this within the k8s-infra call.

Justin will be working on an artifact promoter, if a staging bucket is enough considering it doesn't have guaranteed long retention than the PR is fine, if we want this to have longer retention it should get a manually created prod bucket until we have binary promotion working.

**Comment by spiffxp** [comment]
/approve cancel
Somehow I thought this was an ask for GCR images and a forward looking statement for GCS for binary artifacts.  Let me know what needs to be done here.

**Comment by justinsb** [comment]
I'm not sure what's needed here; we'd like a staging project for etcdadm like other k-sigs projects.

We understand that there isn't yet a fully baked promotion process for binary artifacts (though I'm working on that); when there is though we would like to take advantage of that.  etcdadm will benefit from container distribution in the meantime, along with having the staging locations to get builds going.

File: groups/groups.yaml [hunk 1]
```diff
@@ -839,6 +839,17 @@ groups:
       - jingyih@google.com
       - yczhou@google.com
 
+  - email-id: k8s-infra-staging-etcadm@kubernetes.io
```
Comment (spiffxp) [inline groups/groups.yaml:842]:
```suggestion
  - email-id: k8s-infra-staging-etcdadm@kubernetes.io
```

**Comment by spiffxp** [comment]
(I ran the scripts yesterday, forgot to comment here)

**Pull request merged**